test/cli.py

```diff
@@ -193,25 +214,25 @@ def test_001_cli_single(self):
 
         assert_success_of_builds([test_script])
 
-    def test_002_cli_search_dir(self):
+    def test_002_search_multiple(self):
 
         # Test the first model in the corpus
-        test_script = list(test_scripts_dot_py.keys())[0]
+        test_scripts = list(test_scripts_dot_py.keys())
 
         testargs = [
             "benchit",
             "benchmark",
-            test_script,
-            "-s",
-            corpus_dir,
+            os.path.join(corpus_dir, test_scripts[0]),
+            os.path.join(corpus_dir, test_scripts[1]),
             "--build-only",
             "--cache-dir",
             cache_dir,
         ]
         with patch.object(sys, "argv", testargs):
             benchitcli()
 
-        assert_success_of_builds([test_script])
+        assert_success_of_builds([test_scripts[0]])
+        assert_success_of_builds([test_scripts[1]])
```

Comment:
```suggestion
        assert_success_of_builds(test_scripts)
```

`assert_success_of_builds()`  takes a list of input scripts as input

Reply:
Accepted suggestion!